sources/packages/backend/apps/queue-consumers/src/services/cas-supplier/cas-evaluation-result-processor/cas-evaluation-result-processor.ts

```diff
@@ -25,4 +28,48 @@ export abstract class CASEvaluationResultProcessor {
     evaluationResult: CASEvaluationResult,
     summary: ProcessSummary,
   ): Promise<ProcessorResult>;
+
+  /**
+   * Process the errors that occurred during processing.
+   * @param studentSupplier student supplier information from SIMS.
+   * @param summary current process log.
+   * @param error errors that occurred during processing.
+   * @param auditUserId audit user id.
+   * @param partialSupplier partial supplier information.ß
+   * @returns processor result.
+   */
+  async processBadRequestErrors(
+    studentSupplier: StudentSupplierToProcess,
+    summary: ProcessSummary,
+    error: string[],
+    auditUserId: number,
+    partialSupplier?: Partial<CASSupplier>,
+  ): Promise<ProcessorResult> {
+    summary.warn("A known error occurred during processing.");
+    const now = new Date();
+    const auditUser = { id: auditUserId } as User;
+    try {
+      await this.casSupplierRepo.update(
+        {
+          id: studentSupplier.casSupplierID,
+        },
+        {
+          ...partialSupplier,
+          supplierStatus: SupplierStatus.ManualIntervention,
+          supplierStatusUpdatedOn: now,
+          isValid: false,
+          updatedAt: now,
+          modifier: auditUser,
+          errors: error,
+        },
+      );
+      summary.info("Set supplier status to Manual Intervention due to error.");
```

Comment:
"Manual Intervention" in logs can be changed to use the enum `SupplierStatus.ManualIntervention`.